packages/blocks/video/package.json

```diff
@@ -22,7 +22,7 @@
     "@blockprotocol/graph": "0.0.8"
   },
   "devDependencies": {
-    "block-scripts": "0.0.11",
+    "block-scripts": "0.0.12",
```

Comment:
I guess we’ll need to release a new version of this package once https://github.com/blockprotocol/blockprotocol/pull/394 is updated and merged.

Reply:
Addressed in [181f517](https://github.com/hashintel/hash/pull/683/commits/181f51795af046bb72bbdcacb2e4efb7e2e2526b)